modules/table/src/integrationTest/java/org/apache/ignite/internal/disaster/ItDisasterRecoveryReconfigurationTest.java

```diff
@@ -156,7 +156,9 @@ void setUp(TestInfo testInfo) throws Exception {
 
         ZoneParams zoneParams = testMethod.getAnnotation(ZoneParams.class);
 
-        startNodesInParallel(IntStream.range(INITIAL_NODES, zoneParams.nodes()).toArray());
+        IntStream.range(INITIAL_NODES, zoneParams.nodes()).forEach(i -> cluster.startNode(i));
+        // TODO: Fails with "Race operations took too long"
```

Comment:
According to our guidelines it's not valid to use TODO without a ticket link. Please use https://issues.apache.org/jira/browse/IGNITE-22818